src/Private/Context.ps1

```diff
@@ -814,6 +821,18 @@ function Set-PodeServerConfiguration {
     if ([long]$Configuration.Request.BodySize -gt 0) {
         $Context.Server.Request.BodySize = [long]$Configuration.Request.BodySize
     }
+
+    if ($Configuration.DefaultFolders) {
+        if ($Configuration.DefaultFolders.Public) {
+            $Context.Server.DefaultFolders.Public = $Configuration.DefaultFolders.Public
+        }
+        if ($Configuration.DefaultFolders.View) {
+            $Context.Server.DefaultFolders.View = $Configuration.DefaultFolders.View
+        }
```

Comment:
Just spotted,  the 3 `.View` refs should be `.Views` :)

Reply:
My bad.
This is the major problem with Powershell.
Nothing tells you that what you wrote is rubbish